ncrypt/crypt_gpgme.c

```diff
@@ -1409,6 +1409,42 @@ static void print_smime_keyinfo(const char *msg, gpgme_signature_t sig,
   }
 }
 
+/**
+ * show_one_recipient - Show information about one encryption recipient
+ * @param state State to write to
+ * @param r     Encryption recipient
+ */
+static void show_one_recipient(struct State *state, gpgme_recipient_t r)
+{
+  const char *algo = gpgme_pubkey_algo_name(r->pubkey_algo);
+  if (!algo)
+    algo = "?";
+
+  state_puts(state, "Recipient: ");
+  state_puts(state, algo);
+  state_puts(state, " key, ID ");
+  state_puts(state, r->keyid);
+  state_puts(state, "\n");
```

Comment:
this needs to be a `state_putf(_("Recipient %s ...` or whatever it's spelled as but otherwise sure

Reply:
Thanks!  Will do.